dts/x86/quark_se_c1000_devboard.fixup

```diff
@@ -0,0 +1,9 @@
+#define CONFIG_UART_QMSI_0_BAUDRATE	INTEL_QMSI_UART_B0002000_CURRENT_SPEED
+#define CONFIG_UART_QMSI_0_NAME		INTEL_QMSI_UART_B0002000_LABEL
```

Comment:
what is this cryptic define using B0002000? where is INTEL_QMSI_UART_B0002000_LABEL defined?